alembic/versions/700e63da6309_airdroptables.py

```diff
@@ -0,0 +1,119 @@
+"""AirdropTables
+
+Revision ID: 700e63da6309
+Revises: 
+Create Date: 2021-09-24 16:20:35.212780
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = '700e63da6309'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('airdrop',
+    sa.Column('row_id', sa.INTEGER(), autoincrement=False, nullable=False),
+    sa.Column('address', sa.VARCHAR(length=50), nullable=False),
+    sa.Column('org_name', sa.VARCHAR(length=256), nullable=False),
+    sa.Column('token_name', sa.VARCHAR(length=50), nullable=False),
```

Comment:
Lets increase token name to 128 characters

Reply:
Updated in the latest commit @raamb